nix/haskell.nix

```diff
@@ -58,18 +58,25 @@ let
       # Add dependencies
       {
         packages.cardano-wallet-byron.components.tests = {
+          # # Only run integration tests on non-PR jobsets. Note that
+          # # the master branch jobset will just re-use the cached Bors
+          # # staging build and test results.
+          # cardano-node-integration.doCheck = !isHydraPRJobset;
+
+          # fixme: test suite disabled - they are timing out
+          cardano-node-integration.doCheck = false;
+
           # provide cardano-node command to test suites
           cardano-node-integration.build-tools = [ pkgs.cardano-node ];
```

Comment:
The renaming was bi-directional though. `cardano-wallet-jormungandr:integration` became `cardano-wallet-jormungandr:jormungandr-integration`, but `cardano-wallet-byron:cardano-node-integration` became `cardano-wallet-byron:integration` ... 

Sorry, I didn't thought these were used as references in the nix pipeline.

Reply:
I first made the fix before #1508 was merged then needed to fix it again.

It's likely that if you rename cabal components then any build code that refers to them will need updating.